src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
```

Comment:
This code is just moved from ProBattleUtils and not being refactored at this point.